index.html

```diff
@@ -5,9 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Jobiz</title>
-
     <link rel="stylesheet" href="/src/assets/css/global.css">
+    <script async type="module" src="src/js/localstorage.js"></script> <!-- async so the localstorageTest will have priority-->
```

Comment:
We don't need to import it here. We don't use any of those functions in this page. We will only need those functions in JS, not HTML files.

Reply:
Done.